lib/suite.js

```diff
@@ -108,6 +110,25 @@ Suite.prototype.timeout = function(ms) {
 };
 
 /**
+ * Set number of times to retry a failed test.
+ *
+ * @api private
+ * @param {number|string} n
+ * @return {Suite|number} for chaining
+ */
+Suite.prototype.retries = function(n) {
+  if (!arguments.length) {
+    return this._retries;
+  }
+  if (n.toString() === '0') {
+    this._retries = -1;
```

Comment:
hmm doesn't really matter IMO, as long as `_currentRetry > _retries`. It's more like an indicator.